haskell/src/Monocle/Backend/Index.hs

```diff
@@ -294,10 +321,14 @@ ensureConfigIndex = do
   createIndex configIndex ConfigIndexMapping
   (currentVersion, currentConfig) <- getConfigVersion
 
-  when (currentVersion == ConfigVersion 0) $ traverseWorkspace upgradeConfigV1 conf
+  when (currentVersion == ConfigVersion 0) $ do
+    traverseWorkspace upgradeConfigV1 conf
+    traverseWorkspace upgradeConfigV2 conf
```

Comment:
If this gets too complicated to maintain, I think we could define upgrade procedures like this:

```haskell
upgradeConfig :: [(ConfigVersion, TenantM ())]
upgradeConfig = [
    (ConfigVersion 1, do
     -- ensure crawler updated_at
     undefined
  ), (ConfigVersion 2, do
     --  add cached author
     undefined
  )]
```

So that we can automatically run the migration that are above the current version. With something like `traverse (\(version, procedure) -> when (version < currentVersion) procedure) upgradeConfig`